builders/tools/sidecar.md

```diff
@@ -109,10 +109,8 @@ RESPONSE JSON Block Object:
             |--nonce: 
             |--args
                 |--transaction
-                    |--nonce
-                    |--gasPrice
-                    |--gasLimit
-                    |--signature
+                    |--transaction type
```

Comment:
i'm not really familiar with sidecar but does this have a space in it? seems like other fields use camel case so just wasn't sure

Reply:
good point. this field is where "legacy", "eip1559" or "eip2930" would be, I'll put it in brackets to be consistent.